src/std/hashmap.md

```diff
@@ -24,3 +24,15 @@ fn main() {
     }
 }
 ```
+
+
+<details>
+
+* The HashMap is not defined in the prelude and needs to be brought into scope.
+* In a hashmap, types with the Copy trait (`i32`) are copied into the hashmap, and owned values (`String`) are moved into the hashmap.
+* Try the following to shortcuts. The first line will see if a book is an the hashmap and if not return an alternative value. The second line will insert the alternative value in the hashmap if the book is not found.
```

Comment:
I'm not sure what you mean by "Try the following to shortcuts."

"in the hashmap" rather than "an the hashmap"